theano/tensor/nlinalg.py

```diff
@@ -179,6 +185,10 @@ class ExtractDiag(Op):
     __props__ = ("view",)
 
     def __init__(self, view=False):
+        warnings.warn("DeprecationWarning: theano.tensor.nlinalg.ExtractDiag"
```

Comment:
@nouiz It says ExtractDiag works on the GPU but I can't find its GPU implementation.